api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -276,18 +276,18 @@ message ExternalProcessor {
   // The default value is 5000 milliseconds (5 seconds) if not specified.
   google.protobuf.Duration deferred_close_timeout = 19;
 
-  // [#not-implemented-hide:]
   // Send body to the side stream server once it arrives without waiting for the header response from that server.
   // It only works for STREAMED body processing mode. For any other body processing modes, it is ignored.
   //
   // The server has two options upon receiving a header request:
   // 1. Instant Response: Send the header response as soon as the header request is received.
```

Comment:
It is rendered as below. is this desired? You probably can fix it in next PR.

![image](https://github.com/user-attachments/assets/f7c62ca7-33cc-440e-9a27-8782997731f5)

Reply:
As discussed offline, we will follow up on it in a future PR.